bin/supervisor/src/flags/supervisor.rs

```diff
@@ -67,6 +78,64 @@ impl SupervisorArgs {
         })?;
         Ok(dependency_set)
     }
+
+    /// initialise and return the rollup config set.
```

Comment:
```suggestion
    /// Initialise and return the rollup config set.
```

Reply:
Fixed this in the [source PR](https://github.com/op-rs/kona/pull/1953)